base_classes/NXroot.nxdl.xml

```diff
@@ -46,15 +46,39 @@
     <attribute name="file_update_time" type="NX_DATE_TIME">
         <doc>Date and time of last file change at close</doc>
     </attribute>
+    <attribute name="NeXus_repository">
+        <doc>
+            A repository containing the application definitions
+            used for creating this file.
+            If the NeXus_version attribute contains a commit distance and hash
+            this should refer to this repository.
+        </doc>
+    </attribute>
     <attribute name="NeXus_version">
         <doc>
-            Version of NeXus API used in writing the file.
+            Version of NeXus definitions used in writing the file.
+            This may either be a date based version tag of the form `vYYYY.MM`
```

Comment:
Include a link to https://github.com/nexusformat/definitions/releases as a list of possible version tags.

Reply:
done